packages/common/test/customCrypto.spec.ts

```diff
@@ -24,11 +23,14 @@ describe('[Common]: Custom Crypto', () => {
     return msg
   }
 
-  const customEcSign = (_msg: Uint8Array, _pk: Uint8Array): ECDSASignature => {
+  const customEcSign = (
+    _msg: Uint8Array,
+    _pk: Uint8Array,
+  ): Pick<ReturnType<typeof secp256k1.sign>, 'recovery' | 'r' | 's'> => {
```

Comment:
Maybe also here: ok from TypeScript/Node.js compatibility perspective?

Reply:
have also confirmed that these fancy types do not cause an issue in Node 23 with `node --conditions=typescript --experimental-strip-types...`